remediation/self-node-remediation/faq.md

```diff
@@ -45,7 +45,7 @@ Self Node Remediation has 3 CRDs:
 * SelfNodeRemediationTemplate - automatically created by the operator and should be referenced in the MHC/NHC CR. This is currently empty but we might be used to pass some remediation configuration.
 
 ## Does Self Node Remediation work on its own as a standalone operator?
-Self Node Remediation is a remediation/fencing system. It doesn’t provide any health detection system.
+Self Node Remediation is a remediation/fencing system. It doesn't provide any health detection system.
```

Comment:
What is the difference in here? I can see these are two signs `  vs ' , but aren't they identical?

Reply:
This was an automatic fix. I think that the tool consider the first not an apostrophe :shrug:

Reply:
A quick search confirmed what you say, that backtick and single quote are equivalent, but it is not suggested to mix them. Either use one or the other.

Reply:
Which tool have you been using? Grammarly or similar?

Reply:
Vim spell :sweat_smile: 
Anyway, using the backtick is accepted (interestingly enough, it comes from old computer character sets, which lacks of single quote). Grammarly doesn't complain, but it should be used coherently.